doc/platforms/cc26x0-cc13x0.md

```diff
@@ -4,18 +4,18 @@
 
 **Before you proceed with reading this guide, make sure you have read and understood this section.**
 
-This platform/CPU is in the process of being renamed:
+This platform/CPU was renamed in 2018:
 
-* Current name `srf06-cc26xx`: Used in releases v4.0, v4.1 and v4.2. Used in the pre-release `release-4.2` branch. Used in branch `develop` before 24 October 2018.
-  * Platform code under `arch/platform/srf06-cc26xx`
-  * CPU code under `arch/cpu/cc26xx-cc13xx`
-  * Platform-specific examples under `examples/platform-specific/cc26xx`
-* New name `cc26x0-cc13x0`: Used in branch `develop` as of 24 October 2018.
+* Current name `cc26x0-cc13x0`: Used in release v4.3 and later.
   * Platform code under `arch/platform/cc26x0-cc13x0`
   * CPU code under `arch/cpu/cc26x0-cc13x0`
   * Platform-specific examples under `examples/platform-specific/cc26x0-cc13x0`
+* Legacy name `srf06-cc26xx`: Used in releases v4.0, v4.1 and v4.2. Used in the pre-release `release-4.2` branch. Used in branch `develop` before 24 October 2018.
+  * Platform code under `arch/platform/srf06-cc26xx`
+  * CPU code under `arch/cpu/cc26xx-cc13xx`
+  * Platform-specific examples under `examples/platform-specific/cc26xx`
 
-While reading the guide below, you may have to use a different platform name depending on the version of Contiki-NG that you are using. The name `srf06-cc26xx` will be obsoleted in Release v4.3.
+While reading the guide below, you may have to use a different platform name depending on the version of Contiki-NG that you are using.
```

Comment:
I think the text will be easier to read if the documentation is for the current version, and the history of how the port was named is omitted. If someone cares deeply about a 5 year old release, they can read the documentation for that release (which is available on readthedocs).

Reply:
I agree with you on this.

However - I tried now to find doc. for such old releases but it wasn't immediately obvious. We did not have readthedocs like we do today, so those pages are mostly empty. Seems like one would need to navigate the commits in the wiki repo and based on dates find the correct-ish one (since there are no tags). Or am I missing something?